package.json

```diff
@@ -3,6 +3,7 @@
   "private": true,
   "version": "0.0.0",
   "type": "module",
+  "packageManager": "pnpm@7.32.2",
```

Comment:
I prefer to use pnpm 8

Reply:
But your lockfile is on pnpm 7. Lockfile should be updated first